urllib3/contrib/pyopenssl.py

```diff
@@ -217,7 +217,10 @@ def shutdown(self):
 
     def close(self):
         if self._makefile_refs < 1:
-            return self.connection.close()
+            try:
+                return self.connection.close()
+            except OpenSSL.SSL.Error:
+                return
```

Comment:
I think with a body as short as this one `pass` or `return` are about equally clear. I don't object to the explicit indication that we return `None` in some cases. I think this is 50/50, so I'm inclined to just leave this as-is.